packages/shared/lib/core/ledger/enums/ledger-error.enum.ts

```diff
@@ -5,3 +5,12 @@ export enum LedgerError {
     Locked = 'ledger locked',
     Transport = 'transport error',
 }
+
+export enum LedgerErrorCode {
+    DeviceLocked = '0x6b0c',
+    DisconnectedDevice = 'DisconnectedDevice',
+    NoAppOpen = '0x6511',
+    NoDevice = 'NoDevice',
+    UpdateRequired = '0x6e00',
```

Comment:
I'm not sure if this one is correct, because it seems to be thrown when another app is open as well.

Reply:
Yeah true, although we can rely on `ledgerNanoStatus/ledgerConnectionState` to know what our connection status is.